src/applications/applications.service.ts

```
@@ -329,6 +338,152 @@ export class ApplicationsService {
+    let total = 0;
+
+    for (const rule of benefitDefinition.benefitCalculationRules || []) {
+      let amount = 0;
+
+      switch (rule.type) {
+        case "fixed":
+          amount = rule.fixedValue || 0;
+          break;
+
+        case "lookup":
+          const inputVal = applicationData[rule.inputFields[0]];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)